File: eco-core/core-plugin/src/main/java/com/willfp/eco/spigot/config/DataYml.java
package com.willfp.eco.spigot.config;

import com.willfp.eco.core.config.BaseConfig;
import com.willfp.eco.spigot.EcoSpigotPlugin;
import org.jetbrains.annotations.NotNull;

public class DataYml extends BaseConfig {
    /**
     * Init data.yml.
     *
     * @param plugin EcoSpigotPlugin.
     */
    public DataYml(@NotNull final EcoSpigotPlugin plugin) {
        super("data", false, plugin);
    }
}

File: eco-core/core-plugin/src/main/kotlin/com/willfp/eco/internal/spigot/data/DataYml.kt
package com.willfp.eco.internal.spigot.data

import com.willfp.eco.core.config.BaseConfig
import com.willfp.eco.core.config.ConfigType
import com.willfp.eco.internal.spigot.EcoSpigotPlugin

class DataYml(
    plugin: EcoSpigotPlugin
) : BaseConfig(
    "data",
    plugin,
    false,
    ConfigType.YAML,
    false
)
